Source/Core/Core/IOS/IOS.cpp

```diff
@@ -361,7 +362,10 @@ void Kernel::AddStaticDevices()
   AddDevice(std::make_unique<Device::USB_KBD>(*this, "/dev/usb/kbd"));
   AddDevice(std::make_unique<Device::SDIOSlot0>(*this, "/dev/sdio/slot0"));
   AddDevice(std::make_unique<Device::Stub>(*this, "/dev/sdio/slot1"));
-  AddDevice(std::make_unique<Device::USB_HIDv4>(*this, "/dev/usb/hid"));
+  if ((ios_title_id & 0xFF) == 59)
```

Comment:
Hiding it in a `SupportsHIDv5` method only moves the problem, but what about using `GetVersion` instead? Besides, `AddStaticDevices` is a member of `Kernel`, which already knows its `title_id`.